extract_data.sh

```diff
@@ -1,72 +1,23 @@
-query_tool-popularity() { ## query tool-popularity [months|24]: Most run tools by month
-	handle_help "$@" <<-EOF
-		See most popular tools by month
-		    $ ./gxadmin query tool-popularity 1
-		              tool_id          |   month    | count
-		    ---------------------------+------------+-------
-		     circos                    | 2019-02-01 |    20
-		     upload1                   | 2019-02-01 |    12
-		     require_format            | 2019-02-01 |     9
-		     circos_gc_skew            | 2019-02-01 |     7
-		     circos_wiggle_to_scatter  | 2019-02-01 |     3
-		     test_history_sanitization | 2019-02-01 |     2
-		     circos_interval_to_tile   | 2019-02-01 |     1
-		     __SET_METADATA__          | 2019-02-01 |     1
-		    (8 rows)
-	EOF
+#!/bin/bash
+# Create dir if not existing
+mkdir -p data/
 
-	fields="count=2"
-	tags="tool_id=0;month=1"
-
-	months=${1:-24}
-
-	read -r -d '' QUERY <<-EOF
+# Run the queries
+psql > data/tool-popularity.tsv <<-EOF
+	COPY (
 		SELECT
 			tool_id,
-			date_trunc('month', create_time AT TIME ZONE 'UTC')::date as month,
+			date_trunc('day', create_time AT TIME ZONE 'UTC')::date as day,
```

Comment:
Since I am using the monthly usage data for tools, I will update it to `month` and not as `day`.

Reply:
I thought you swapped to day since you'd requested that? Huh, ok :)

Reply:
I tried with `day` but the learning time went up by a large margin. I am using non-linear regressor for that and the running time depends on the amount of data. Therefore, I switched back to `month`.